charts/nd-common/templates/_istio.tpl

```diff
@@ -79,6 +79,25 @@ proxy.istio.io/overrides: >-
   } 
 {{- end }}
 
+{{/*
```

Comment:
Nit. We make careful use of `{{- ` syntax to make sure we don't add newlines to the rendered output when they aren't necessary. For example, this code currently makes this diff:

```diff
 metadata:
   name: simple-app
   labels:
-    helm.sh/chart: simple-app-1.6.0
+    helm.sh/chart: simple-app-1.6.1
     app.kubernetes.io/version: "latest"
     app.kubernetes.io/managed-by: Helm
     tags.datadoghq.com/service: "simple-app"
@@ -242,6 +242,8 @@
               }
             ]
           }
+        
+        
         ad.datadoghq.com/simple-app.check_names: '["prometheus"]'
         ad.datadoghq.com/simple-app.init_configs: '[{}]'
         ad.datadoghq.com/simple-app.instances: |-
@@ -261,7 +263,7 @@
             }
           ]
```

Notice the new-lines? If you change this to `{{- /*` then the diff is:

```diff
 metadata:
   name: simple-app
   labels:
-    helm.sh/chart: simple-app-1.6.0
+    helm.sh/chart: simple-app-1.6.1
     app.kubernetes.io/version: "latest"
     app.kubernetes.io/managed-by: Helm
     tags.datadoghq.com/service: "simple-app"
@@ -261,7 +261,7 @@
             }
           ]
       labels:
-        helm.sh/chart: simple-app-1.6.0
+        helm.sh/chart: simple-app-1.6.1
         app.kubernetes.io/version: "latest"
         app.kubernetes.io/managed-by: Helm
         tags.datadoghq.com/service: "simple-app"
```